src/main/java/sg/ncl/domain/ExceptionState.java

```diff
@@ -1,25 +1,58 @@
 package sg.ncl.domain;
 
+import lombok.Getter;
+import sg.ncl.exceptions.EmailAlreadyExistsException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'sg.ncl.exceptions.EmailAlreadyExistsException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)